### PR TITLE
Adding com_users reset view test.

### PR DESCRIPTION
**Pull request by charvimehradu**
Pull Request for Issue # .
Adding com_users reset view test.

### CODE REVIEW

**Comment by charvimehradu** [comment]
@laoneo , can you review this PR and let me know if it looks okay?

File: tests/System/integration/site/components/com_users/Reset.cy.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+        cy.get('#system-message-container').should('contain.text', 'If the email address you entered is registered on this site you will shortly receive an email with a link to reset the password for your account.');
+      });
+  });
+
+  it('testing reset form for test user through menu item', () => {
+    cy.db_createUser({
+      name: 'test user', username: 'test', email: 'test@example.com', password: '098f6bcd4621d373cade4e832627b4f6',
+    })
+      .then(() => cy.db_createMenuItem({ title: 'Automated test reset', link: 'index.php?option=com_users&view=reset&id=1&layout=default', path: '?option=com_users&view=reset&id=1&layout=default' }))
```
Comment (laoneo) [inline tests/System/integration/site/components/com_users/Reset.cy.js]: Can you remove the whole path attribute, it is not required anymore. Why do you have in the link attribute id=1?

File: tests/System/integration/site/components/com_users/Reset.cy.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+describe('Test the reset view', () => {
+  it('can display the reset view for test user', () => {
+    cy.db_createUser({
+      name: 'test user', username: 'test', email: 'test@example.com', password: '098f6bcd4621d373cade4e832627b4f6',
```
Comment (laoneo) [inline tests/System/integration/site/components/com_users/Reset.cy.js]:
```suggestion
      email: 'test@example.com',
```

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**